src/AuthReg.cc

```diff
@@ -30,18 +31,18 @@
 void
 Auth::Init()
 {
-    debugs(29,DBG_IMPORTANT,"Startup: Initializing Authentication Schemes ...");
+    debugs(29, Important(68), "Startup: Initializing Authentication Schemes ...");
 #if HAVE_AUTH_MODULE_BASIC
     static const char *basic_type = Auth::Basic::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << basic_type << "'");
+    debugs(29, 2, "Startup: Initialized Authentication Scheme '" << basic_type << "'");
 #endif
 #if HAVE_AUTH_MODULE_DIGEST
     static const char *digest_type = Auth::Digest::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << digest_type << "'");
+    debugs(29, 2, "Startup: Initialized Authentication Scheme '" << digest_type << "'");
 #endif
 #if HAVE_AUTH_MODULE_NEGOTIATE
     static const char *negotiate_type = Auth::Negotiate::Scheme::GetInstance()->type();
-    debugs(29,DBG_IMPORTANT,"Startup: Initialized Authentication Scheme '" << negotiate_type << "'");
+    debugs(29, 2, "Startup: Initialized Authentication Scheme '" << negotiate_type << "'");
```

Comment:
BTW, if you want me to remove somewhat misleading `Startup: ` prefixes from the four messages above, please let me know, and I will do that in this PR. These messages are not specific to Squid startup. Auth::Init() is also called on reconfigurations...

Reply:
Sure :)